dtlstransport.go

```diff
@@ -45,6 +45,7 @@ type DTLSTransport struct {
 	srtcpSession  atomic.Value
 	srtpEndpoint  *mux.Endpoint
 	srtcpEndpoint *mux.Endpoint
+	srtpReady     chan interface{}
```

Comment:
Seems used only for closing:
```suggestion
	srtpReady     chan struct{}
```

Reply:
Fixed!